service/local.ml

```diff
@@ -34,6 +34,16 @@ let repo =
   @@ Arg.info ~doc:"The directory containing the .git subdirectory." ~docv:"DIR"
        []
 
+let submission_solver_service =
+  Arg.value
+  @@ Arg.opt Arg.(some Capnp_rpc_unix.sturdy_uri) None
+  @@ Arg.info
+       ~doc:
+         "The solve.cap/submission.cap file for the build scheduler service \
```

Comment:
Reading fast I thought you meant a subdirectory instead of `solve.cap` or `submission.cap`. You should pick one or the other, for clarity. Maybe use what is in the solver-service documentation if any?